### DISCUSSION TITLE
Are my archives failing suddenly?

### DISCUSSION THREAD

**Discussion by gerroon** [ArchiveBox/ArchiveBox#1303] [Q&A] [answered]
See the image, all those three did not achieve, how would I know what the issues are? Is there a solid page that I can test for ground truth? 

If someone can try one of these pages that would be great as well.

![image](https://github.com/ArchiveBox/ArchiveBox/assets/8519469/97d1731f-1764-4683-af94-834fe91ff575)

**Comment by pirate** [comment, marked as answer]
Are you on the latest `archivebox/archivebox:dev` build? Please post your full `docker compose run archivebox version` output. The builds are updated often and I can't provide support without that full unredacted version info to know exactly what setup you're running with. We also need it at the top of every new issues/discussions (even if you're on "the most recent version" or have posted that info in other threads), we need every new discussions/question/bug report to stay visibly associated to + searchable by the specific version they're about.

You can always check the `data/logs/error.log` file or in the `UI Admin > Log` page to see details on why stuff is failing. Or you can click the leftmost date column on any snapshot to go to its admin page, then scroll down to the bottom to see the log entries specifically for that snapshot. 
<img width="900" alt="image" src="https://github.com/ArchiveBox/ArchiveBox/assets/511499/a5a0642a-167f-4973-9e8a-8cfc4ac9b652">


Try increasing your `archivebox config --set TIMEOUT=120` to see if it's a slowness/timeout issue, as that would affect all methods.

You can try archiving these URLs:
- https://sqlsync.dev/posts/stop-building-databases
- https://klittlepage.com/2015/01/21/why-i-always-hit-the-crosswalk-button

Output should look like this:
<img width="1419" alt="image" src="https://github.com/ArchiveBox/ArchiveBox/assets/511499/f8e3af9c-2733-418d-baf6-6f15abfaa9f5">

https://demo.archivebox.io/archive/1703282516.986707/index.html
